app/controllers/idv/review_controller.rb

```diff
@@ -50,9 +50,12 @@ def new
 
     def create
       irs_attempts_api_tracker.idv_password_entered(success: true)
+      log_reproof_event if Profile.find_by(id: idv_session&.profile_id)&.active?
```

Comment:
The Profile model defines reproofing as:

```ruby
is_reproof = Profile.find_by(user_id: user_id, active: true)
```

But this is in the `activate` method and is only true if you call it before it goes and sets `active: true`, which foiled my plan to pull this out into a `reproofed?` method.

This line _should_ be able to be written as:

```ruby
log_reproof_event if idv_session&.profile&.active?
```

But, this ran into a lulzy issue, where `idv_session.profile` was causing `profile` to get populated before expected, breaking a test.

https://github.com/18F/identity-idp/blob/main/app/services/idv/session.rb#L91-L93

I didn't really want to remove the memoization there because it's potentially used in many places.